sql/migrate/migrate.go

```diff
@@ -59,6 +59,19 @@ type (
 	}
 )
 
+// Statement format the change's command
+func (p Plan) Statement(c *Change) string {
```

Comment:
sqltool contains different formatter. The plan is agnostic to the actual format.

Reply:
Maybe make it a template function?